src/amo/components/StaticPages/About/index.js

```diff
@@ -0,0 +1,111 @@
+          <p>
+            {i18n.gettext('There are several ways to find out the latest news from the world of add-ons:')}
+          </p>
+          <ul>
+            <li dangerouslySetInnerHTML={sanitizeHTML(blogText, ['a'])} />
+            <li>
+              {i18n.gettext('We often post news, tips, and tricks to our Twitter account,')}&nbsp;
+              <a href={'http://twitter.com/mozamo'}>{i18n.gettext('mozamo')}</a>
+            </li>
+            <li dangerouslySetInnerHTML={sanitizeHTML(forumText, ['a'])} />
+          </ul>
+        </section>
+        <section>
+          <h2>{i18n.gettext('This sounds great! How can I get involved?')}</h2>
+          <p>{i18n.gettext(`There are plenty of ways to get involved. If you're on the technical side:`)}
+            <ul>
+              <li>
+                <Link to={'/developers'}>{i18n.gettext('Make your own add-on')}</Link>.&nbsp;
+                {i18n.gettext('We provide free hosting and update services and can help you reach a large audience of users.')}
+              </li>
+              <li>
+                {i18n.gettext('If you have add-on development experience,')}&nbsp;
+                <a href="https://wiki.mozilla.org/Add-ons/Reviewers/Applying">{i18n.gettext('become an Add-on Reviewer')}</a>!&nbsp;
+                {i18n.gettext('Our reviewers are add-on fans with a technical background who review add-ons for code quality and stability.')}
+              </li>
+              <li dangerouslySetInnerHTML={sanitizeHTML(helpText, ['a'])} />
+            </ul>
+          </p>
+          <p>{i18n.gettext(`If you're interested in add-ons but not quite as technical, there are still ways to help:`)}</p>
+          <ul>
+            <li>{i18n.gettext('Tell your friends! Let people know which add-ons you use.')}</li>
+            <li>{i18n.gettext('Participate in our')}&nbsp;
+              <a href="https://discourse.mozilla-community.org/c/add-ons">{i18n.gettext('forums')}</a>.
+            </li>
+            <li>
+              {i18n.gettext('Review add-ons on the site. Add-on authors are more likely to improve their add-ons and write new ones when they know people appreciate their work.')}
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h2>{i18n.gettext('I have a question')}</h2>
+          <p>{i18n.gettext('A good place to start is our')}&nbsp;
+            <Link to="/faq"><abbr title="{i18n.gettext('Frequently Asked Questions')}">{i18n.gettext('FAQ')}</abbr></Link>.&nbsp;
+            {i18n.gettext(`If you don't find an answer there, you can`)}&nbsp;<a href="https://discourse.mozilla-community.org/c/add-ons">{i18n.gettext('ask on our forums')}</a>.
+          </p>
+          <p>{i18n.gettext('If you really need to contact someone from the Mozilla team, please see our')}&nbsp;
+            <Link to="/developers/docs/policies/contact">{i18n.gettext('contact information')}</Link> {i18n.gettext('page.')}
```

Comment:
You could save a redirect and just point to `https://developer.mozilla.org/en-US/Add-ons#Contact_us` for this - also avoids the complication with `Link`.